prime-router/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml

```diff
@@ -30,10 +30,11 @@ elements:
     value: [ '""' ]
     hl7Spec: [ /PATIENT_RESULT/PATIENT/PID-5-7 ]
 
-  - name: ak-patient-race-system
-    condition: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding.system.contains("v3-NullFlavor")'
-    value: [ '"NULLFL"' ]
-    hl7Spec: [ /PATIENT_RESULT/PATIENT/PID-10-3 ]
+  - name: ak-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
```

Comment:
Is there supposed to be quotes around this? The other ones don't seem to have it.